tree/master/cloud/src/solution/Microsoft.Legal.MatterCenter.Web/Controllers/ConfigController.cs

```diff
@@ -147,7 +147,15 @@ IHostingEnvironment hostingEnvironment
                 #endregion
                 var configResultsVM = await configRepository.GetConfigurationsAsync(filter);
                 CreateConfig(configResultsVM, "uiconfigforspo.js", true);
-                return matterCenterServiceFunctions.ServiceResponse(configResultsVM, (int)HttpStatusCode.OK);
+                configRepository.UploadConfigFileToSPO(Path.Combine(hostingEnvironment.WebRootPath, "app/config.js"), generalSettings.CentralRepositoryUrl);
+                var genericResponse = new GenericResponseVM

```

Comment:
@AISDUG  After successfully adding the files to the catalog site collection doc library, the response type to client will be just success or failure. No need to send the complete json object with all the config settings